openpype/lib/file_transaction.py

```diff
@@ -82,6 +84,14 @@ def add(self, src, dst, mode=MODE_COPY):
                         src, dst))
                 return
             else:
+                if not self._allow_queue_replacements:
+                    raise RuntimeError(
```

Comment:
I would use different exception e.g. `ValueError` or custom. And maybe catch it in integrate.py and reraise as `KnownPublishError`.

Reply:
I'll go for `DuplicateDestinationError` because it's not necessary that it's the same file. Anyway, thanks!